src/pe/symbol.rs

```diff
@@ -412,6 +412,7 @@ pub struct AuxSectionDefinition {
 }
 
 /// A COFF symbol table.
+// TODO: #[derive(Pwrite)] produce unparseable tokens
```

Comment:
I can't remember did this get fixed in scroll upstream?

Reply:
I don't think so, there's a PR to do so right?

Reply:
I'm not seeing one; it's ok not super relevant for your PR, was just curious